lib/reline.rb

```diff
@@ -366,18 +366,14 @@ def readline(prompt = '', add_hist = false)
       rescue Errno::EIO
         # Maybe the I/O has been closed.
       rescue StandardError => e
```

Comment:
Do we still need to rescue `StandardError` separately after this?

Reply:
Nice catch :+1:
No need. updated

Also removed `rescue Exception` which is added in #404, looks like only to run finalize and deprep